### DISCUSSION TITLE
StartStreamTranscription on @aws-sdk/client-chime

### DISCUSSION THREAD

**Discussion by brunokunace** [aws/aws-sdk-js-v3#4632] [Q&A]
Hi guys, i trying to use the live transcription of chime. I tested with default configurations and works fine.

I have an API with Nestjs with this code for test:

```js
      const client = new Chime({})

      const transcriptionConfiguration: TranscriptionConfiguration = {
        EngineTranscribeSettings: {
          LanguageCode: 'en-US'
        }
      }

      return client.startMeetingTranscription({
        MeetingId: meetingId,
        TranscriptionConfiguration: transcriptionConfiguration,
      }, (err, data) => {
        console.log('err', err);
        console.log('data', data);
      })
      
 ```
 
 But i would pass others parameters in TranscriptionConfiguration, like `identify-language` and `show-speaker-label`, found here: 
 [StartStreamTranscription](https://docs.aws.amazon.com/transcribe/latest/APIReference/API_streaming_StartStreamTranscription.html) and not have in this sdk.
 
 The real doubts are: 
 1) Can i use this parameters in live transcription?
 2) I need to use the API and not use this sdk for my case?
 3) Why the language is required if have a identify language feature?

**Comment by yenfryherrerafeliz** [comment]
Hi @brunokunace, the linked documentation references a different client which is not chime. The client this documentation refers to is the TranscribeStreamingClient, which you can for sure pass those parameters. [Here](https://github.com/aws/aws-sdk-js-v3/blob/fb5df75b6ed6a417aa9ee19d862eefe83d7286ed/clients/client-transcribe-streaming/src/models/models_0.ts#L1854) is the definition of the parameters that we can pass to this specific operation.

I hope this helps. Please let me know if you have any questions.

Thanks!
